pom.xml

```diff
@@ -230,6 +231,7 @@
                         <arg>--add-exports </arg>
                         <arg>java.base/jdk.internal.logger=openjceplus</arg>
                     </compilerArgs>
+                    <fork>true</fork>
```

Comment:
Is there any reason why we need to run this compiler plugin in another JVM process?

Reply:
If we want to let some plugins to run in parallel, perhaps we can add one for `maven-surefire-plugin`.

Reply:
It doesn't have to do with running things in parallel.